feat(node): Add more detailed OTEL docs (#10872)
getsentry/sentry-docs: docs/platforms/javascript/common/opentelemetry/index.mdx

---
title: OpenTelemetry Support
description: "Using OpenTelemetry with Sentry."
sidebar_order: 7000
supported:
  - javascript.nextjs
  - javascript.node
  - javascript.aws-lambda
  - javascript.azure-functions
  - javascript.connect
  - javascript.express
  - javascript.fastify
  - javascript.gcp-functions
  - javascript.hapi
  - javascript.koa
  - javascript.nestjs
  - javascript.sveltekit
  - javascript.astro
  - javascript.remix
notSupported:
  - javascript
---

> The Sentry SDK uses [OpenTelemetry](https://opentelemetry.io/) under the hood. This means that any OpenTelemetry instrumentation that emits spans will automatically be picked up by Sentry - no further configuration needed! To start using Sentry and OpenTelemetry together, set up <PlatformLink to='/tracing'>Tracing and Performance Monitoring with your Sentry SDK</PlatformLink>.

Reviewer: This index page is a bit confusing. I'm not sure how to best do this but this page should clearly answer why as a user I would need to read this section of the docs (and most likely, why not).

For instance: The first sentences tell me that I don't have to do anything. Otel and Sentry works. But then I should follow a link to "use Sentry and Otel together"?

I think we should convey that the SDK uses Otel _for tracing and spans_. And maybe just drop the "Sentry and OpenTelemetry together" because it can mean a lot of things. Something along the lines of:

```suggestion
The Sentry SDK uses [OpenTelemetry](https://opentelemetry.io/) under the hood for tracing and span creation. This means that any OpenTelemetry instrumentation that emits spans will automatically be picked up by Sentry - no further configuration needed! 

To get started with tracing, follow the guide to set up <PlatformLink to='/tracing'>Tracing and Performance Monitoring with your Sentry SDK</PlatformLink>.
```

Reviewer: Ah I saw you just updated the paragraph below which makes it a bit clearer. I'd probably still reword the first paragraph but feel free to do as you see fit.

Author: the thing is that it is not correct to say the SDK uses otel for tracing and spans, otel is also used for context isolation etc 😬 so I would avoid to say that otel is needed only for tracing/spans or something like this 🤔 

I'll still try to reword this a bit to make this clearer, hopefully...!

---
title: OpenTelemetry Support
description: "Using OpenTelemetry with Sentry."
sidebar_order: 7000
supported:
  - javascript.nextjs
  - javascript.node
  - javascript.aws-lambda
  - javascript.azure-functions
  - javascript.connect
  - javascript.express
  - javascript.fastify
  - javascript.gcp-functions
  - javascript.hapi
  - javascript.koa
  - javascript.nestjs
  - javascript.sveltekit
  - javascript.astro
  - javascript.remix
notSupported:
  - javascript
---

The Sentry SDK uses [OpenTelemetry](https://opentelemetry.io/) under the hood. This means that any OpenTelemetry instrumentation that emits spans will automatically be picked up by Sentry - no further configuration needed!

> To start capturing traces and spans emitted from OpenTelemetry, set up <PlatformLink to='/tracing'>Tracing and Performance Monitoring with your Sentry SDK</PlatformLink>. If you do not use tracing, Sentry still connects to OpenTelemetry under the hood to ensure that context isolation and trace propagation works correctly.

Author: @Lms24 I changed this a bit to this, do you like this better/is this clearer to you?

> To start capturing traces and spans emitted from OpenTelemetry, set up <PlatformLink to='/tracing'>Tracing and Performance Monitoring with your Sentry SDK</PlatformLink>. If you do not use tracing, Sentry still connects to OpenTelemetry under the hood to ensure that context isolation and trace propagation works correctly.

Reviewer: l: sorry for the nit; I think our docs guidelines somewhere tell us to use "don't" over "do not"  

```suggestion
To start capturing traces and spans emitted from OpenTelemetry, set up <PlatformLink to='/tracing'>Tracing and Performance Monitoring with your Sentry SDK</PlatformLink>. If you don't use tracing, Sentry still connects to OpenTelemetry under the hood to ensure that context isolation and trace propagation works correctly.
```

Author: yeah totally correct, thank you for checking everything :D

> To start capturing traces and spans emitted from OpenTelemetry, set up <PlatformLink to='/tracing'>Tracing and Performance Monitoring with your Sentry SDK</PlatformLink>. If you do not use tracing, Sentry still connects to OpenTelemetry under the hood to ensure that context isolation and trace propagation works correctly.

Reviewer: maybe we just omit "OpenTelemetry" here?
```suggestion
To start capturing traces and spans, set up <PlatformLink to='/tracing'>Tracing and Performance Monitoring with your Sentry SDK</PlatformLink>. If you do not use tracing, Sentry still connects to OpenTelemetry under the hood to ensure that context isolation and trace propagation works correctly.
```
